openapi/components/schemas/CreditMemos/CreditMemoProperties.yaml

```
@@ -0,0 +1,172 @@
+id:
+  description: Credit memo ID.
+  readOnly: true
+  allOf:
+    - $ref: ../ResourceId.yaml
+number:
+  description: An auto-incrementing number based on the sequence of credit memos for any particular customer.
+  readOnly: true
+  type: integer
+allocations:
+  type: object
+  description: Allocations reduce unused amount of the credit memo.
+  properties:
+    transactions:
+      type: array
+      description: |-
+        List of transactions (typically refunds, credits, and chargebacks) that are allocated to reduce the unused amount of a credit memo.
+      items:
+        type: object
+        properties:
+          transactionId:
+            description: ID of the transaction that the credit memo is allocated to.
```

nitpick: suggestion to avoid ending in with a preposition
```suggestion
            description: ID of the transaction to which the credit memo is allocated.
```